### PR TITLE
Add read only query support.

### PR DESCRIPTION
**Pull request by mikewilson-dd**
Just quickly adding in support for the QUERY_RO command.

### CODE REVIEW

**Comment by mikewilson-dd** [comment]
@filipecosta90 Hey, sorry to ping you directly -- not sure how active this repo is, but I could really use this feature and it's pretty simple.

**Review by filipecosta90 — APPROVED** [review]

**Comment by filipecosta90** [comment]
> @filipecosta90 Hey, sorry to ping you directly -- not sure how active this repo is, but I could really use this feature and it's pretty simple.

Hi there @mikewilson-dd , thank you for submitting the PR and pushing it forward. LVGTM. Asking @swilly22 or @jeffreylovitz for an expert opinion on graph :)

File: graph.go [hunk 1]
```diff
@@ -108,6 +108,17 @@ func (g *Graph) Query(q string) (*QueryResult, error) {
 	return QueryResultNew(g, r)
 }
 
+// QueryRO executes a read only query against the graph.
+func (g *Graph) QueryRO(q string) (*QueryResult, error) {
```
Comment (swilly22) [inline graph.go:112]: Looks good, but please rename from `QueryRO` to `RO_QUERY` to be aligned with the command name

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: graph.go (hunk 1 above)
Comment (mikewilson-dd) [reply graph.go:112]: Ack, will do. To clarify, `RO_QUERY` or `ROQuery`?

Comment (mikewilson-dd) [reply graph.go]: @swilly22 I've renamed this to ROQuery to be more in line with the current `Query` function. Let me know if you agree/disagree and I can merge if you're okay with this.

Comment (swilly22) [reply graph.go]: Can't we go with `RO_Query` ?

Comment (mikewilson-dd) [reply graph.go]: Sure. PR update incoming in a bit.

File: graph.go [hunk 2]
```diff
@@ -108,6 +108,17 @@ func (g *Graph) Query(q string) (*QueryResult, error) {
 	return QueryResultNew(g, r)
 }
 
+// RO_Query executes a read only query against the graph.
+func (g *Graph) RO_Query(q string) (*QueryResult, error) {
```
[the remark this answers is not in the document]
Comment (mikewilson-dd) [reply graph.go:112]: This is at @swilly22 recommendation.

**Comment by filipecosta90** [comment]
@mikewilson-dd after discussing the naming with @swilly22 we've agreed that is best to keep the language standards and go with your suggestion of `RoQuery`, making the following `func (g *Graph) ROQuery(q string) (*QueryResult, error)` signature. 
I've reverted that specific change from the latest commit and we're good to go :+1: 
Thank you for pushing this forward!

**Comment by mikewilson-dd** [comment]
Thanks all!

**Pull request merged**